### PR TITLE
MODLD-457: Store Identifiers of Authority records in Linked Data data…

### PR DESCRIPTION
**Pull request by AndreiBordak**
Dictionary PR: https://github.com/FOLIO-EIS/lib-linked-data-dictionary/pull/80
Marc4ld PR: https://github.com/FOLIO-EIS/lib-linked-data-marc4ld/pull/147
Linked Data PR: https://github.com/FOLIO-EIS/mod-linked-data/pull/318

### CODE REVIEW

**Review by pkjacob — APPROVED** [review]

File: src/main/java/org/folio/marc4ld/service/marc2ld/authority/MarcAuthority2ldMapperImpl.java [hunk 1]
```diff
@@ -63,6 +71,26 @@ private Resource fillResource(Resource resource, org.marc4j.marc.Record marcReco
...
+      .filter(dataField -> dataField.getTag().equals(FIELD_UUID))
+      .findFirst().map(
+        metadata -> new FolioMetadata()
+          .setSource(MARC)
+          .setInventoryId(getSubfieldValue(metadata, SUBFIELD_INVENTORY_ID))
+          .setSrsId(getSubfieldValue(metadata, S))
+      );
+  }
+
+  private String getSubfieldValue(DataField dataField, char code) {
```
Comment (askhat-abishev) [inline src/main/java/org/folio/marc4ld/service/marc2ld/authority/MarcAuthority2ldMapperImpl.java:89]: there is ```getSubfieldValue``` method in ```MarcUtil``` class
how about moving this code there?

File: src/test/java/org/folio/marc4ld/mapper/field999/Marc2Authority999IT.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    var marc = loadResourceAsString("authority/999/marc_999.jsonl");
+
+    // when
+    var resources = marcAuthorityToResources(marc);
+
+    //then
+    var actualResource = resources.stream().findFirst();
+    assertThat(actualResource).isPresent();
+    var folioMetadata = actualResource.get().getFolioMetadata();
+    assertThat(folioMetadata.getInventoryId()).isEqualTo("2165ef4b-001f-46b3-a60e-52bcdeb3d5a1");
```
Comment (askhat-abishev) [inline src/test/java/org/folio/marc4ld/mapper/field999/Marc2Authority999IT.java:23]: minor
you can use ```assertEquals(...)```

**Review by askhat-abishev — APPROVED** [review]

**Review by PBobylev — APPROVED** [review]

File: src/test/java/org/folio/marc4ld/mapper/field999/Marc2Authority999IT.java (hunk 2 above)
Comment (AndreiBordak) [reply src/test/java/org/folio/marc4ld/mapper/field999/Marc2Authority999IT.java:23]: fixed.

File: src/main/java/org/folio/marc4ld/service/marc2ld/authority/MarcAuthority2ldMapperImpl.java (hunk 1 above)
Comment (AndreiBordak) [reply src/main/java/org/folio/marc4ld/service/marc2ld/authority/MarcAuthority2ldMapperImpl.java:89]: done.

**Pull request merged**